evap/results/tools.py

```diff
@@ -91,10 +92,6 @@ def average(self):
     def has_answers(self):
         return self.is_published and any(count != 0 for count in self.counts)
 
-    @property
-    def is_published(self):
-        return self.counts is not None
```

Comment:
I liked this more.

We agreed that it might still work similarly

Reply:
Can you review again pls?